tailwind.config.js

```diff
@@ -4,7 +4,13 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {colors:{
+      'regal-blue': '#1D8BCC'
+    },
+    backgroundImage:{
+      'lpBackground': "url('./LandingPage/lpBackground.svg')",
```

Comment:
Explanatory keys and variable names

Reply:
Do you mean I should re-construct the variable keys and names?